wiegand/api.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+from RPi.GPIO import *
+import time
+import json
+
+class Wiegand26:
+    #Reads pin numbers for input pins from JSON
+    #Facility number is also set from JSON
+    def __init__(self):
+        config = {}
+        with open('config.json', 'r') as config_file:
```

Comment:
what did you choose to read the config from json in the first place? disk input/output is expensive. and you only have 3 config options anyway. it's fine to just initialize these like so:
```
self.data0 = 15
self.data1 = 13
self.facility = 32
```
also, if we're gonna hardwire it, the config can be hardcoded since it doesn't make sense to be configurable anymore.